api/openapi-spec/swagger.json

```diff
@@ -10356,6 +10356,9 @@
      "imagePullPolicy": {
       "type": "string"
      },
+     "launcherRuntimeClass": {
```

Comment:
how about `defaultRuntimeClassName` like mentioned in https://github.com/kubevirt/kubevirt/issues/5744#issuecomment-856154191? I wounder if we should pick this up on the virt-handler daemonset too.